### DISCUSSION TITLE
issues that can be closed

### DISCUSSION THREAD

**Discussion by spnethw** [elfmz/far2l#2296] [General]
@elfmz 

Позакрывать бы все неактуальные issues — что уже пофикшено/реализовано,  что не планируется или вообще не воспроизводится.
Эта пачка из трёх сотен тикетов и мешает просматривать список, в поисках того, что ещё надо сделать, и впечатление о проекте портит.

Вот эти, например:

~https://github.com/elfmz/far2l/issues/2010~
~https://github.com/elfmz/far2l/issues/1933~
~https://github.com/elfmz/far2l/issues/1917~
~https://github.com/elfmz/far2l/issues/1869~
~https://github.com/elfmz/far2l/issues/1833~

UPD: 
~https://github.com/elfmz/far2l/issues/2476~

**Comment by akruphi** [comment]
Также возможно доросли до закрытия (не во всех уверен, что их точно можно закрывать):
~#2264~
~#2116~
~#2021~
~#2017~
~#1972~
#1971
#1945
~#1916~
~#1870~
~#1785~
#1716
~#1615~
~#1508~
~#1488~
#1190
~#1131~
~#1000~
#968
~#877~
~#874~
~#850~
#666
#579
~#572~
~#306~
~#272~
#263
#255
~#208~
~#87~
~#6~

UPD: ~#1985~

 UPD: Закрытые 04.12.2024 перечернул

**Comment by elfmz** [comment]
позакрывал многие, но не все
